workspaces/ui-v2/src/lib/__tests/diff-helpers/__snapshots__/shape-interpretations.test.ts.snap

```diff
@@ -6376,9 +6376,11 @@ Object {
     Object {
       "SetResponseBodyShape": Object {
         "bodyDescriptor": Object {
-          "httpContentType": "application/json",
-          "isRemoved": false,
-          "shapeId": "shape_0",
+          "ShapedBodyDescriptor": Object {
```

Comment:
This should have been removed, so the previous version is correct. It smells like someone overwrote changes in a merge?

Reply:
Oh i don't think I rebuilt the workspace